middlewares/upload.js

```diff
@@ -0,0 +1,22 @@
+const multer = require('multer');
+const path = require('path');
+const uploadDir = path.join(process.cwd(), 'tmp');
```

Comment:
tmp public i public/avatars nie istnieje na repo (git nie przechowuje folderów tylko same pliki z ich lokacją). Przydało by się przy starcie servera sprawdzić czy te 3 ścieżki istnieją i je tworzyć jeśli nie. (zrobisz to biblioteką "fs")